VocaDbModel/DataContracts/Venues/VenueWithEventsContract.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections.Generic;
+using System.Linq;
+using VocaDb.Model.DataContracts.ReleaseEvents;
+using VocaDb.Model.Domain.Globalization;
+using VocaDb.Model.Domain.ReleaseEvents;
+using VocaDb.Model.Domain.Venues;
+
+namespace VocaDb.Model.DataContracts.Venues {
+
+	public class VenueWithEventsContract : VenueContract {

```

Comment:
How about, if you add Events (as optional field) to VenueForApiContract, then you shouldn't need this contract anymore, right? I've been moving towards using those "Api" contracts with optional fields whenever possible.